src/components/ExtraPage/Farmer.js

```diff
@@ -22,10 +22,13 @@ function Farmer(){
         return (
           <>
             <Col md={4} xs={12} key={obj.tree_id} className="FarmerTreeCard" onClick={(e) => {
-                addToSelectedTree(obj)
+                addToSelectedTree(obj);
                 alert(`You successfully selected a ${obj.tree_name} tree!`)
                 }}>
-              <Card className='TreeCard' id={index}>
+              <Card className='TreeCard' id={index} style={selectedTree.tree_id == obj.tree_id ? {
+                borderWidth: 2,
```

Comment:
Added a style attribute that (using a conditional ternary operator) when the tree is the selected one, it will show a green border. I haven't tried it because I don't know where to find the feature in the frontend.